pkg/controllers/user/logging/configsyncer/configsyncer.go

```diff
@@ -137,7 +137,12 @@ func (s *ConfigSyncer) syncClusterConfig(clusterLoggings []*mgmtv3.ClusterLoggin
 	secretName := loggingconfig.RancherLoggingConfigSecretName()
 	namespace := loggingconfig.LoggingNamespace
 
-	buf, err := s.configGenerator.GenerateClusterLoggingConfig(clusterLoggings[0], systemProjectID)
+	var clusterLogging *mgmtv3.ClusterLogging
+	if len(clusterLoggings) != 0 {
```

Comment:
If there is nothing for input, it should be return earlier. I would like to fix this in line 111. `If len(clusterLoggings) = 0`, `syncClusterConfig` won't be called.

Reply:
when the (clusterLoggings) == 0, it need to update the secret to empty config

Reply:
I got it now. We need to have a empty `[]byte` to update the configuration secret.